.github/workflows/auto-pr-from-master-into-releases.yml

```diff
@@ -0,0 +1,58 @@
+name: PR changes on master into release branches
+env:
+    SOURCE_BRANCH: master
+on:
+    # This workflow should trigger when changes are pushed to master.
+    # We expect this will happen when PRs into master are merged and also as
+    # part of the automated bugfix release process.
+    push:
+        branches:
+            # ${{ env.SOURCE_BRANCH }} does not seem to work here.
```

Comment:
I did, yes!  I would prefer to use `${{ env.SOURCE_BRANCH }}` instead of statically defining `master`, and so I left the comment because it will help me to remember why I'm defining the branch twice.